### ISSUE TITLE
Certification on PowerBI.com

### ISSUE THREAD

**Issue by corny359** [weiweicui/PowerBI-Flowmap#22]
Hi,
thanks for this great visual!
Is there a chance for you to certify this visual in the AppSource? My company actually blocks all uncertified visuals on PowerBI.com.
I don't know how much effort this is, but I would really appreciate.

**Comment by cyongt** [comment]
Yes, that would be really great if you could certify this very useful visuals in Microsoft Power BI AppSource! Microsoft certification is also needed from my company. Thanks a lot!

**Comment by OrrinEdenfield** [comment]
I've not reviewed everything in the code yet, but I'm not seeing anything preventing this visual from getting certified per the guidelines published [here](https://learn.microsoft.com/en-us/power-bi/developer/visuals/power-bi-custom-visuals-certified). Does anyone know of any technical limitations with getting this visual certified?
